galaxy/main/signals/handlers.py

```diff
@@ -37,21 +36,29 @@
 
 @receiver(user_logged_in)
 def user_logged_in_handler(request, user, **kwargs):
+    social = auth_models.SocialAccount.objects.get(
+        provider=constants.PROVIDER_GITHUB.lower(), user=user)
+    user.avatar_url = social.extra_data.get('avatar_url')
```

Comment:
This is fine for now. However, in the future, I anticipate we'll have a page where the user can manage their profile, and update their `avatar_url`. When that becomes true, we should only set the `user.avatar_url` when it is empty.

Reply:
I'm totally agree.